core/modules/ccontrol/A4UserQueryFactory.cc

```diff
@@ -34,29 +37,48 @@
 
 #include "parser/QSMySqlListener.h"
 #include "query/SelectStmt.h"
-
+#include "util/IterableFormatter.h"
 
 
 // antlr4 C++ runtime seems to require that we use namespace antlr4; trying to use it with classes
 // results in at least one error where the `tree` decl assumes that it's already operating in the
 // antlr4 namespace. I may be wrong about this though; it needs research (before merge to master).
 // If I'm right we should see about fixing the antlr4 cpp runtime & issuing a PR to them.
 using namespace antlr4;
+using namespace std;
```

Comment:
Are we really supposed to do this? This may open a whole "can of worms".

Reply:
The comment above also applies to:
using namespace antlr4;

Reply:
in a .cc it's ok. verboten in .h files.

Reply:
Ok, as long as you feel its safe 👍